src/conrad_relaycard/card.py

```diff
@@ -36,6 +36,7 @@ def _get_serial_port(self, port: str | None = None) -> serial.Serial:
             )
 
         if not self._serial_port.is_open:
+            self._serial_port.close()
```

Comment:
Is it safe to call close() on a already closed serial device?

Reply:
Good question. I change it to try-close. Also forgot to put the close on every raise. That was the intention from beginning